File: codeforces/617A_elephant.go
package main

import (
	"fmt"
)

func main(){
	var x, cociente, resultado int
	fmt.Scanf("%d", &x)
	i := 5
	for i > 0 {
		cociente = x / i
        resultado = resultado + cociente
        x = x % i
		i = i - 1
	}
	fmt.Println(resultado)
}

File: codeforces/617A_elephant.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int x, resto, cociente, resultado = 0;
    cin >> x;
    for(int i=5; i >= 1; i--){
        cociente = x / i;
        resultado = resultado + cociente;
        x = x % i;
    }
    cout << resultado << "\n";
}
